packages/components/src/Header.tsx

```diff
@@ -20,104 +21,141 @@ export const Header = ({
     setAlertMessage: () => void
     title: string
 }) => {
+    const [isButtonHovered, setIsButtonHovered] = useState(false)
```

Comment:
Let's keep like in the Figma for now, but I just noticed now there is no button that leads to the `/bids` page. Will work something out with Lucía.

Reply:
The project title lead now to `/bids` and the cura logo leads to `/`

Reply:
https://github.com/curaOS/source/blob/2766b8d9415b24aaef0547bd3965896bb1fe616a/packages/components/src/Header.tsx#L95

Reply:
I saw that but it might be confusing. Fine for now, but will probably change slightly the design to make a link to bids fit :)